Telefonica/mistica-ios
# Popover improvements (align with figma specs)

## 🎟️ **Jira ticket**
https://jira.tid.es/browse/SMARTWIFI-7203

## 🥅 **What's the goal?**
Adapt Mistica to be able to reproduce Figma design for GamingOn Services List: popover should take the width of the parent minus 32pt and target should be subtitleLabel.

The width of the popover is given by the specifications found in Figma: https://www.figma.com/file/gmcy7hwB9WXxqT05jBnA3W/🔸-Popover-Specs?type=design&node-id=0-1&mode=design&t=V6XUGxKShA0L0AnT-0


## 🚧 **How do we do it?**
- Mark labels as public
- Update popover `setupConstraints()` method

## 🧪 **How can I verify this?**
1. Compile Mistica catalog
2. Remove subtitle (leaving default title, which is short)
3. Verify popover width takes 100% - 32pt.

## 🏑 **AppCenter build**
